test/unit/specs/components/staking/TabStakingParameters.spec.js

```diff
@@ -0,0 +1,144 @@
+import {shallowMount, createLocalVue} from "@vue/test-utils"
+import TabStakingParameters from "renderer/components/staking/TabStakingParameters"
+import lcdClientMock from "renderer/connectors/lcdClientMock.js"
```

Comment:
Can be done separately, but recently I was trying to remove this import every time I touch a test file